### PR TITLE
Add and update i18nReady: false pages

### PR DESCRIPTION
**Pull request by xMohamd**
update i18nReady: false pages and include it if it doesn't exist 
this applies for files within `src/content/docs/en/`

See #6549 for details

### CODE REVIEW

File: src/content/docs/en/guides/upgrade-to/v1.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Legacy v0.x Upgrade Guide 
 description: Archived guide documenting changes between pre-v1 versions of Astro
-i18nReady: false
+i18nReady: true
```
Comment (liruifengv) [inline src/content/docs/en/guides/upgrade-to/v1.mdx:4]:
```suggestion
i18nReady: false
```

The page is outdate. I don't think we need to translate it.

File: src/content/docs/en/guides/content.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: >-
   Astro is a perfect choice for your content-focused site: blogs, marketing
   sites, portfolios, and more! Author your content directly in your project, or
   connect your CMS of choice.
-i18nReady: false
+i18nReady: true
```
Comment (liruifengv) [inline src/content/docs/en/guides/content.mdx:7]: `content.mdx` content doesn't seem to be included in the documentation site?

**Comment by liruifengv** [comment]
Most of the pages in the cms have been translated, will this cause lunaria to mark them `outdated` again?

File: src/content/docs/en/guides/content.mdx (hunk 2 above)
Comment (xMohamd) [reply src/content/docs/en/guides/content.mdx:7]: @liruifengv i think you are right, but I'll wait for @Yan-Thomas to confirm it before changing it

**Comment by Yan-Thomas** [comment]
> Most of the pages in the cms have been translated, will this cause lunaria to mark them outdated again? @Yan-Thomas
Not really, this PR will be marked as `i18nIgnore` so this doesn't happen.

**Comment by xMohamd** [comment]
> > Most of the pages in the cms have been translated, will this cause lunaria to mark them outdated again? @Yan-Thomas
> 
> Not really, this PR will be marked as `i18nIgnore` so this doesn't happen.

Sorry, but I didn't understand what you meant.

File: src/content/docs/en/guides/internationalization.mdx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Internationalization
 description: Learn how to use i18n routing in Astro.
-i18nReady: false
+i18nReady: true
```
Comment (sarah11918) [inline src/content/docs/en/guides/internationalization.mdx:4]: Oh no, how was _this_ page ever not translated?  :sweat_smile: 

This page will have updates on Thursday with the minor release though, so I'm going to say we should hold off marking this one just yet.
```suggestion
i18nReady: false
```

File: src/content/docs/en/guides/content.mdx (hunk 2 above)
Comment (sarah11918) [inline src/content/docs/en/guides/content.mdx:7]: Also, this one is not really discoverable yet, and I might want to use it for other purposes. Great call liruifengv!
```suggestion
i18nReady: false
```

File: src/content/docs/en/guides/upgrade-to/v1.mdx (hunk 1 above)
Comment (sarah11918) [reply src/content/docs/en/guides/upgrade-to/v1.mdx]: Agreed, this page is *really* old.

**Review by sarah11918 — COMMENTED** [review]
Reviewed for the pages that should not be translated at the moment!

**Comment by sarah11918** [comment]
@xMohamd The `i18nIgnore` is a keyword we use in PR titles or commits to indicate that the changes made here should **not** trigger to mark any translated files as needing an update. None of the changes made are things the translators need to worry about updating on their end, so we have a keyword so that our tracking system knows that our entire i18n system and contributors can safely ignore the changes from this PR!

**Pull request merged**